db/db.go

```diff
@@ -2523,10 +2523,6 @@ func GetAddressWithdrawals(address []byte, limit uint64, offset uint64) ([]*type
 		limit = 100
 	}
 
-	if limit > (offset + 100) {
-		limit = offset + 100
-	}
```

Comment:
I also noticed such weird settings for APIs.
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/d6eda8928fcf92ffdfac1923ea75c1327e9be767/handlers/api.go#L562
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/d6eda8928fcf92ffdfac1923ea75c1327e9be767/handlers/api.go#L570
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/d6eda8928fcf92ffdfac1923ea75c1327e9be767/handlers/api.go#L1550

Should they immediately be changed or is this not relevant enough for the issue?

Reply:
I have created BIDS-1665 for that.